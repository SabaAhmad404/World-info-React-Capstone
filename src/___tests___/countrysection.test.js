import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from '@cfaester/enzyme-adapter-react-18';
import CountrySection from '../components/countrysection';

configure({ adapter: new Adapter() });

describe('Country information', () => {
  it('renders correctly components', () => {
    const wrapper = shallow(
      <CountrySection
        name="Pakistan"
        population="220892331"
        flag="https://flagcdn.com/ke.svg"
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('renders correctly components', () => {
    const wrapper = shallow(
      <CountrySection
        name="India"
        population="1380004385"
        flag="https://flagcdn.com/ke.svg"
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
