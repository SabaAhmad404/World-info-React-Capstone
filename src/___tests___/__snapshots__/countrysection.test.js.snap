// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Country information renders correctly components 1`] = `ShallowWrapper {}`;

exports[`Country information renders correctly components 2`] = `ShallowWrapper {}`;
